examples/README.md

```diff
@@ -247,49 +247,49 @@ Intel® Neural Compressor validated examples with multiple compression technique
     <td>*SSD ResNet50 V1</td>
     <td>Object Detection</td>
     <td>Post-Training Static Quantization</td>
-    <td><a href="./examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq">pb</a> / <a href="./examples/tensorflow/object_detection/tensorflow_models/ssd_resnet50_v1/quantization/ptq">ckpt</a></td>
```

Comment:
Those issues have been fixed in mater branch, you can revert the change and rebase master.